README.md

```diff
@@ -60,15 +67,51 @@ The pipeline performs long read QC, assembly, end repair, polishing, contig rota
 
 ```bash
 git clone https://github.com/jmtsuji/rotary.git
-conda env create -n rotary --file=rotary/envs/enviroment.yaml
+conda env create -n rotary --file=rotary/enviroment.yaml
+conda activate rotary
+cd rotary
+pip install --editable .
+```
+
+### 2a. Run One Sample
+
+```bash
+mkdir output_dir
+mkdir rotary_db_dir
+
+cd output_dir
+
+rotary run_one -l s1_long.fastq.gz -r1 s1_R1.fastq.gz -r2 s1_R2.fastq.gz -d ../rotary_db_dir
+```
+**Note**: If you are using older nanopore flow cells you should stop the run, modify the config file 
+(see **Advanced Usage** below) and restart the run using the `rotary run` command.
+
+### 2b. Run Multiple Samples
+
+Rotary can target a directory containing numerous FASTQ files derived from various samples.
+It effortlessly organizes these files into sets corresponding to each sample and constructs a project 
```

Comment:
Maybe OK to remove "effortlessly" for conciseness?

Reply:
This is what happens when you use chatgpt to rewrite you stuff for proof editing purposes lol. It tends to use flowery sales language by default. I think the original was "automatically" and it changed it to "effortlessly", which is better for sales.

Reply:
I changed it back to automatically.